CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 - xo-server-usage-report v0.7.0
 - xo-server v5.29.0
 - xo-web v5.29.0
+- @xen-orchestra/fs v0.4.0
```

Comment:
Please move it on top because it's a dependency of other packages to help with the (in order) release process :slightly_smiling_face: